Bandwidth.Iris/Model/Csr.cs

```diff
@@ -0,0 +1,146 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml.Serialization;
+
+namespace Bandwidth.Iris.Model
+{
+
+    [XmlType("Csr")]
+    public class Csr : BaseModel
+    {
+
+        [XmlElement("accountId")]
+        public string AccountId { get; set; }
+        public string CustomerOrderId { get; set; }
+        public string AccountNumber { get; set; }
+        public string AccountTelephoneNumber { get; set; }
+        public string EndUserName { get; set; }
+        public string AuthorizingUserName { get; set; }
+        public string CustomerCode { get; set; }
+        public string EndUserPIN { get; set; }
+        public string EndUserPassword { get; set; }
+        public string AddressLine1 { get; set; }
+        public string City { get; set; }
+        public string State { get; set; }
+        public string ZIPCode { get; set; }
+        public string TypeOfService { get; set; }
```

Comment:
I think you need `Status` as well since that's an expected parameter for the update/replace request